ample/util/tm_util.py

```diff
@@ -503,8 +503,8 @@ def _mod_structures(self, model_aln, structure_aln, model_pdb, structure_pdb):
         # Alignment not always identical, let's aim for 90%
         identical = set(_model_data).intersection(set(_structure_data))
         if len(identical) / float(len(_model_data)) < 0.90:
-            msg = "Differing residues in model and structure. Affected PDBs %s - %s"
-            raise RuntimeError(msg % (model_name, structure_name))
+            msg = "Differing residues in model and structure. Affected PDBs %s - %s\n%s\n%s"
```

Comment:
`os.linesep` is probably cleaner than `\n`

Reply:
True, but it's within a string, so I think passing os.linesep in and using another %s would be harder to read.